src/mlpack/methods/ann/loss_functions/KLDivergence.hpp

```diff
@@ -0,0 +1,112 @@
+/**
+ * @file KLDivergence.hpp
+ * @author Dakshit Agrawal
+ *
+ * Definition of the KL Divergence error function.
```

Comment:
Might be a good idea to call this Kullback–Leibler divergence.

Reply:
I have changed the file name. Should I change class name too?

Reply:
I think the class and filename is just fine, it's a pretty common alias, but we should mention at least once Kullback–Leibler divergence in the comments.